codegen/smithy-dafny-codegen-modules/smithy-python/codegen/smithy-python-codegen/src/main/java/software/amazon/smithy/python/codegen/StructureGenerator.java

```diff
@@ -252,7 +252,7 @@ protected boolean isOptionalDefault(MemberShape member) {
             && (target.isDocumentShape() || target.isListShape() || target.isMapShape());
     }
 
-    private void writeClassDocs(boolean isError) {
+    protected void writeClassDocs(boolean isError) {
```

Comment:
Unfortunately, this change means another

`make mvn_local_deploy_polymorph_python_dependencies`

on merging this from main, but I think this is better than duplicating the code.

Reply:
I'm inclined to add `mvn_local_deploy_polymorph_dependencies` as a dependency of all polymorph targets, given it should be cheap and hopefully Gradle-cached anyway.

Reply:
Good idea. I'll include this